scripts/profiler/rtt2socket.py

```diff
@@ -0,0 +1,231 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+from rtt_nordic_config import RttNordicConfig
+import sys
+import logging
+import time
+from pynrfjprog.LowLevel import API
+from pynrfjprog.APIError import APIError
+from enum import Enum
+from stream import Stream
```

Comment:
We may consider naming it ProfilerStream to avoid suggesting that it's a generic stream